### PR TITLE
Tracy/Helpers.php: pid added to CLI [closes #152]

### LINKED ISSUE #152: Add PID to BlueScreen
EasyPick Feature: add process ID (PID) to BlueScreen

[relevant line in source code](https://github.com/nette/tracy/blob/a54ad9a2cd6ffa7c382b0aa58ed7739c263387fa/src/Tracy/Helpers.php#L152)

### PR DESCRIPTION
**Pull request by Northys**

### CODE REVIEW

File: src/Tracy/Helpers.php [hunk 1]
```diff
@@ -149,7 +149,8 @@ public static function getSource()
 				. (isset($_SERVER['HTTP_HOST']) ? $_SERVER['HTTP_HOST'] : '')
 				. $_SERVER['REQUEST_URI'];
 		} else {
-			return empty($_SERVER['argv']) ? 'CLI' : 'CLI: ' . implode(' ', $_SERVER['argv']);
+			$output = 'CLI (pid: ' . getmypid() . ')';
+			return empty($_SERVER['argv']) ? $output : $output . ': ' . implode(' ', $_SERVER['argv']);
 		}
```
Comment (dg) [inline src/Tracy/Helpers.php]: this is imho more readable
```php
return 'CLI (pid: ' . getmypid() . ')' 
   . (empty($_SERVER['argv']) ? '' : ': ' . implode(' ', $_SERVER['argv']));
```

**Comment by Northys** [comment]
@dg better? :)

**Comment by dg** [comment]
Fix indentation and change commit message to something like `Helpers::getSource: shows PID in CLI`

File: src/Tracy/Helpers.php [hunk 2]
```diff
@@ -149,7 +149,8 @@ public static function getSource()
 				. (isset($_SERVER['HTTP_HOST']) ? $_SERVER['HTTP_HOST'] : '')
 				. $_SERVER['REQUEST_URI'];
 		} else {
-			return empty($_SERVER['argv']) ? 'CLI' : 'CLI: ' . implode(' ', $_SERVER['argv']);
+			return 'CLI (PID: ' . getmypid() . ')'
+			. (empty($_SERVER['argv']) ? '' : ': ' . implode(' ', $_SERVER['argv']));
```
Comment (JanTvrdik) [inline src/Tracy/Helpers.php]: The second line should be indented.

Comment (Northys) [inline src/Tracy/Helpers.php]: gonna fix it tonight

**Comment by JanTvrdik** [comment]
ping

**Comment by Northys** [comment]
@JanTvrdik it should be ok, sorry forgot to push
@dg ping

**Comment by Northys** [comment]
thank you for merging :)

**Comment by dg** [comment]
Thanks for PR :-)

**Pull request merged**